packages/pyright-internal/src/tests/harness/fourslash/workspaceEditTestUtils.ts

```diff
@@ -91,6 +91,11 @@ export function verifyDocumentEdits(
                             return false;
                         }
 
+                        if (!actualEdit.textDocument.uri.includes(':')) {
```

Comment:
Under what circumstance would this ever occur? Or are you just being super paranoid here?

Reply:
Just being super paranoid. It's why the test didn't fail before. This ensures we don't make this same mistake again.